Add function of estimating step (#9)
icon-project/icon-rpc-server: docs/icon-json-rpc-v3.md

}
```

## debug_estimateStep

> * Generates and returns an estimates step cost of how much step is necessary to allow the transaction to complete as not being added to the blockchain. Note that the estimate may be more than the actual amount of a step cost used by the transaction for several reasons including node performance.

Reviewer: I think it's better to split the first sentence into two like below. And use just step, not step cost.
```suggestion
* Generates and returns an estimated step of how much step is necessary to allow the transaction to complete. The transaction will not be added to the blockchain. Note that the estimate may be more than the actual amount of step used by the transaction for several reasons including node performance.
```

### Parameters

> * The transaction's information not to have stepLimit and signature keys

Reviewer: ```suggestion
* The transaction information not to have stepLimit and signature
```

| to        | [T_ADDR_EOA](#T_ADDR_EOA) or [T_ADDR_SCORE](#T_ADDR_SCORE) | required | EOA address to receive coins, or SCORE address to execute the transaction. |
| value     | [T_INT](#T_INT)                                            | optional | Amount of ICX coins in loop to transfer. When ommitted, assumes 0. (1 icx = 1 ^ 18 loop) |
| timestamp | [T_INT](#T_INT)                                            | required | Transaction creation time. timestamp is in microsecond.      |
| nid       | [T_INT](#T_INT)                                            | required | Network ID ("0x1" for Mainnet, "0x2" for Testnet, etc)       |
| nonce     | [T_INT](#T_INT)                                            | optional | An arbitrary number used to prevent transaction hash collision. |
| dataType  | [T_DATA_TYPE](#T_DATA_TYPE)                                | optional | Type of data. (call, deploy, or message)                     |
| data      | T_DICT or String                                           | optional | The content of data varies depending on the dataType. See [Parameters - data](#sendtxparameterdata). |

### Returns

> * The amount of an estimated step cost in loop

Reviewer: ```suggestion
* The amount of an estimated step in loop
```
